docs/general/session.md

```diff
@@ -11,12 +11,17 @@ Consequently, a Session is represented as a collection of Logs, Events, and Span
 throughout the Session's duration. Each Session is assigned a unique identifier, which is included as an attribute in
 the Logs, Events, and Spans generated during the Session's lifecycle.
 
+When a session reaches end of life, typically due to user inactivity or session timeout, a new session identifier
+will be assigned. The previous session identifier may be provided by the instrumentation so that telemetry
+backends can link the two sessions.
+
 ## Attributes
 
 <!-- semconv session-id -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `session.id` | string | A unique id to identify a session. | `00112233-4455-6677-8899-aabbccddeeff` | Opt-In |
+| `session.previous_id` | string | The previous `session.id` for this user, when known. | `00112233-4455-6677-8899-aabbccddeeff` | Opt-In |
```

Comment:
Not sure if it's overthinking but do we want to encode both in one id?

For eg., session.id could be an 32 byte identifier, with 16 bytes of it for the previous session. So, a totally new session's id would start with 16 zeroes at the beginning.

Reply:
I think this could cause more problems and I'm not sure if the benefits would be worth it, if I understood correctly we'd have to constrain the format of the `session.id` to always be 32 bytes long, all numbers and we'd always have to make it "previous_id"-aware by having to prepend 16 zeroes to it all the time, even for the cases where knowing about the previous id might not be a requirement, we might also have to create apis to set both ids separately and the sdks would have to merge them later, otherwise it could be a bit error-prone for anybody to manually build the right format of the id as a single attr. From the server side of things, it'd always have to process incoming session ids to split them up and make them easier to read later I believe.

Reply:
@breedx-splk this comment can be resolved. thanks.